assets/src/components/cluster/LabelsAnnotations.tsx

```diff
@@ -34,45 +36,58 @@ export function LabelsAnnotationsRow({
 
 export function LabelsAnnotationsTag({ name, value }: LabelPair) {
   return (
-    <Chip>
+    <Chip size="small">
       {name}
       {value && `: ${value}`}
     </Chip>
   )
 }
 
+function renderLabel(label: Maybe<LabelPair>) {
+  return (
+    <>
+      {label?.name}
+      {label?.value && `: ${label.value}`}
+    </>
+  )
+}
+
 export function LabelsAnnotations({
   metadata: { labels, annotations },
   ...props
 }: {
   metadata: MetadataT
-} & CardProps) {
+  } & CardProps) {
+  const theme = useTheme()
+
   return (
-    <Card
-      padding="xlarge"
-      {...props}
-    >
+    <MetadataCard {...props}>
```

Comment:
The card will be empty if there are no labels and annotations?

Reply:
Yeah, I guess I should hide the card completely in that case.

Reply:
Doesn't render card if there's no data at all.